cmds/core/yes/yes.go

```diff
@@ -7,19 +7,35 @@ package main
 import (
 	"flag"
 	"fmt"
+	"io"
+	"log"
+	"os"
 	"strings"
 )
 
-func main() {
-	flag.Parse()
-	args := flag.Args()
+func runYes(w io.Writer, count uint64, args []string) error {
 	yes := "y"
```

Comment:
in fact I kinda strongly thing it ought tobe ... string

That makes it clear it can be empty

Reply:
done